cmd/thanos/downsample.go

```diff
@@ -346,6 +361,7 @@ func processDownsampling(ctx context.Context, logger log.Logger, bkt objstore.Bu
 
 	level.Info(logger).Log("msg", "downsampled block",
```

Comment:
Might be good to initialize a variable for passed_time above this line so that we don't have to call `time.Since(begin)` multiple times.

Reply:
Agreed